packages/block-library/src/group/edit.js

```diff
@@ -98,11 +97,7 @@ function GroupEdit( {
 	} = attributes;
 
 	// Layout settings.
-	const defaultLayout = useSetting( 'layout' ) || {};
-	const usedLayout = ! layout?.type
-		? { ...defaultLayout, ...layout, type: 'default' }
-		: { ...defaultLayout, ...layout };
-	const { type = 'default' } = usedLayout;
```

Comment:
@tellthemachines This is a change that could be a separate PR. I found that the `useSetting( 'layout' )` call can be completely removed, because the `usedLayout` value is no longer used. Only its `.type` field is used. And the `type` can be calculated without knowing `defaultLayout`. That value always comes from the `layout` object.

`usedLayout` was used until #47477 before usages of `__experimentalLayout`, but today it's redundant.

Reply:
Ah well spotted! Yes, this can be removed. Not too fussed about whether it's done here or in a separate PR, your choice 😄